pkg/tsdb/loki/api.go

```diff
@@ -78,6 +85,8 @@ func makeDataRequest(ctx context.Context, lokiDsUrl string, query lokiQuery) (*h
 		return nil, err
 	}
 
+	addOauthHeader(req, oauthToken)
+
 	// NOTE:
```

Comment:
There is this NOTE about missing `OAuth` data. With this PR, we can remove/adjust this comment, right?

Reply:
good catch! removed.